pkg/querier/astmapper/parallel.go

```diff
@@ -90,7 +87,7 @@ func CanParallelize(node parser.Node) bool {
 		return true
 
 	default:
-		level.Error(util_log.Logger).Log("err", fmt.Sprintf("CanParallel: unhandled node type %T", node))
+		log.Printf("err=\"CanParallel: unhandled node type %T\"", node)
```

Comment:
I have a mixed feeling about it. We're just going from a global logger to another global one. I would keep the previous code until we figure out a better solution. To skip the `faillint` check in this specific case you can add the comment `//lint:ignore faillint`

Reply:
Will do